GeoHealthCheck/app.py

```diff
@@ -977,6 +977,35 @@ def api_probes_avail(resource_type=None, resource_id=None):
     return jsonify(probes)
 
 
+@APP.route('/api/v1.0/runs/<int:resource_id>')
```

Comment:
Fine as is for now, however suggest should we cover a real API later on (soon).

Reply:
Yes, for now the API is on a need-basis, real API is project by itself, so many options and frameworks.